tests/testGetInstcal.py

```diff
@@ -85,9 +85,6 @@ def testInstcal(self):
         self.assertAlmostEqual(weather.getAirPressure(), visit229388_info['weath_airPressure'])
         self.assertAlmostEqual(weather.getHumidity(), visit229388_info['weath_humidity'])
 
-        # Example of metadata which should *not* have been copied from zeroth extension.
-        self.assertNotIn("PROPOSER", exp.getMetadata().paramNames())
```

Comment:
I don't know for DECam, but in general yes -- things like OBS-DATE and EXPTIME tend to be in the PDU.